packages/instant-meilisearch/__tests__/filter.test.ts

```diff
@@ -50,38 +55,6 @@ describe('Instant Meilisearch Browser test', () => {
     expect(hits[0].title).toEqual('Four Rooms')
   })
 
-  test('one string facet on filter without a query', async () => {
-    const response = await searchClient.search<Movies>([
-      {
-        indexName: 'movies',
-        params: {
-          query: '',
-          facetFilters: ['genres:Adventure'],
-        },
-      },
-    ])
-
-    const hits = response.results[0].hits
-    expect(hits.length).toEqual(1)
-    expect(hits[0].title).toEqual('Star Wars')
-  })
-
-  test('one facet on filter with a query', async () => {
-    const response = await searchClient.search<Movies>([
-      {
-        indexName: 'movies',
-        params: {
-          query: 'four',
-          facetFilters: ['genres:Crime'],
-        },
-      },
-    ])
-
-    const hits = response.results[0].hits
-    expect(hits.length).toEqual(2)
-    expect(hits[0].title).toEqual('Four Rooms')
-  })
-
```

Comment:
Why were those tests removed?

Reply:
Because they were added twice, it seems.

https://github.com/meilisearch/meilisearch-js-plugins/blob/f390bc953085c63170775233d59d14b37a87d2cf/packages/instant-meilisearch/__tests__/filter.test.ts#L37-L51

https://github.com/meilisearch/meilisearch-js-plugins/blob/f390bc953085c63170775233d59d14b37a87d2cf/packages/instant-meilisearch/__tests__/filter.test.ts#L69-L83